### ISSUE TITLE
[amd64] raspi repo signature error

### ISSUE THREAD

**Issue by openoms** [raspiblitz/raspiblitz#4029]
```
W: GPG error: http://archive.raspberrypi.org/debian bullseye InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 82B129927FA3303E
E: The repository 'http://archive.raspberrypi.org/debian bullseye InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```
Solved by removing:
```
sudo rm /etc/apt/sources.list.d/raspi.list 
```
so shouldn't be added

**Comment by rootzoll** [comment]
@openoms can this issue be closed for v1.10.0?

**Comment by openoms** [comment]
yes, the solution is merged and working.
